src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useContext, useState } from 'react';
+import cn from 'classnames';
+import { TodoContext } from '../Context';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const { deleteTodo, toggleTodo, updateTodo } = useContext(TodoContext);
+  const { title } = todo;
+
+  const [isEditing, setIsEditing] = useState(false);
+  const [editedTitle, setEditedTitle] = useState(title);
+
+  return (
+    <div data-cy="Todo" className={cn('todo', { completed: todo.completed })}>
+      {/*eslint-disable jsx-a11y/label-has-associated-control */}
+      <label className="todo__status-label">
```

Comment:
```suggestion
      <label className="todo__status-label" aria-label="todo-status">
```
it should fix eslint warning/error